src/components/CalendarMonthGrid.jsx

```diff
@@ -363,7 +367,11 @@ class CalendarMonthGrid extends React.Component {
 CalendarMonthGrid.propTypes = propTypes;
 CalendarMonthGrid.defaultProps = defaultProps;
 
-export default withStyles(({ reactDates: { color, zIndex } }) => ({
+export default withStyles(({
+  reactDates: {
+    color, noScrollBarOnVerticalScrollable, spacing, zIndex,
```

Comment:
each of these needs to be on its own line, if the entire object literal can't fit on one line